doc/users/installing.rst

```diff
@@ -10,20 +10,20 @@ Dependencies
 **Requirements**
 
 These are external packages which you will need to install before
-installing basemap. 
+installing Basemap. 
 
 
-matplotlib 1.0.0 (or later, `download <http://sf.net/projects/matplotlib/>`__)
+Matplotlib 1.0.0 (or later, `download <http://sf.net/projects/matplotlib/>`__)
 
-Python 2.4 (or later, including Python 3)
-    matplotlib requires python 2.4 or later (`download <http://www.python.org/download/>`__)
+Python 2.6 (or later, including Python 3)
+    Matplotlib requires Python 3.5 or later (`download <http://www.python.org/download/>`__)
```

Comment:
Matplotlib 2.2.x will be an LTS branch, and supports py27. Matplotlib v3 hasn't been released yet, which will support a minimum of py35, yes. Should probably think of how to phrase this.

Reply:
How's the latest commit look?